src/Provider/AssetServiceProvider.php

```diff
@@ -36,9 +36,18 @@ function ($app) {
             $fullPath = $app['resources']->getPath('root') . '/' . $fileName;
 
             if (is_readable($fullPath)) {
-                return substr(md5($app['asset.salt'] . (string) filemtime($fullPath)), 0, 10);
+                $path = $fullPath;
             } elseif (is_readable($fileName)) {
-                return substr(md5($app['asset.salt'] . (string) filemtime($fileName)), 0, 10);
+                $path = $fileName;
+            } else {
+                $path = false;
```

Comment:
@GawainLynch I'm not sure I follow.. When I use this, $path already is a full path, that's used for the hash: 

ie: 

```
"/Users/bob/Sites/bolt-origin///extensions/vendor/bobdenotter/seo/assets/seo.css"
"/Users/bob/Sites/bolt-origin///extensions/local/bobdenotter/gridviewfield/assets/handsontable.full.min.css"
"/Users/bob/Sites/bolt-origin///extensions/local/bobdenotter/gridviewfield/assets/handsontable.full.min.js"
"/Users/bob/Sites/bolt-origin///extensions/local/bobdenotter/gridviewfield/assets/start.js"
```

What should be added?